ansible/roles/version/tasks/main.yaml

```
@@ -16,8 +16,9 @@
 #                the build tag.
 - name: determine kubernetes images
   shell: |
-    kubeadm config images list --kubernetes-version='{{ kubernetes_semver }}' --image-repository '{{ k8s_image_registry }}' |
-    sed 's/^\(.\+etcd:\)\(.\+\)-\([0-9]\+\)$/\1{{ "\2-\3" if not kubernetes_build_tag else "v\2"}}/' |
+    kubeadm config images list \
+      --kubernetes-version='{{ kubernetes_semver }}' --image-repository '{{ k8s_image_registry }}' |
+    sed 's/^\(.\+etcd:.\+\)-\([0-9]\+\)$/\1{{ "-\2" if not kubernetes_build_tag }}/' |
```

@faiq Looks like this line is rebased correctly from main. It has a fix for using `v` in version.
https://github.com/mesosphere/konvoy-image-builder/blob/main/ansible/roles/version/tasks/main.yaml#L17
```suggestion
    kubeadm config images list --kubernetes-version='{{ kubernetes_semver }}' --image-repository '{{ k8s_image_registry }}' |
    sed 's/^\(.\+etcd:\)\(.\+\)-\([0-9]\+\)$/\1{{ "\2-\3" if not kubernetes_build_tag else "v\2"}}/' |
```